dotnet/src/IntegrationTests/Connectors/Weaviate/WeaviateMemoryStoreTests.cs

```diff
@@ -0,0 +1,279 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Linq;
```

Comment:
```suggestion
```

Reply:
Done - the merge in https://github.com/microsoft/semantic-kernel/pull/1219/commits/6163a92b80623b2dea886e57defc5c56e349e840 created this issue.